app/views/businessmatching/PsrNumberView.scala.html

```diff
@@ -121,16 +130,20 @@
 
                 @subtitle(section)
 
+                @heading("businessmatching.psr.number.title")
```

Comment:
Might be the case that this content needs to be added for the above case statement too, have you checked what the difference is between this case and the one above?